gradle/newModule.gradle.kts

```diff
@@ -21,7 +21,8 @@ import java.util.Calendar
 abstract class NewModule : DefaultTask() {
 
     private companion object {
-        const val VALID_NAME_REGEX = "^[^\\dA-Z-][a-z]+[a-z-]*\\Z"
+        const val VALID_MODULE_NAME_REGEX = "^[^\\dA-Z-][a-z]+[a-z-]*\\Z"
+        const val VALID_PACKAGE_NAME = "^[a-z]+(\\.[a-z]+)*\\Z"
```

Comment:
Can be also suffixed as `_REGEX` for consistency

Reply:
[8c97f06](https://github.com/splendo/kaluga/pull/523/commits/8c97f063c052dbbc915063a7f7fb6f00262f81a9)